public/app/features/admin/migrate-to-cloud/EmptyState/CallToAction.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
We have a `CallToActionCard` component in grafana/ui, maybe that'll work with some customisation?

Reply:
this one i'm not sure about cause they're so different 🤔 basically just ends up reimplementing this component and passing it as the `callToActionElement` prop which feels a bit icky 😅

Reply:
Tbh I think the current component we have could use some updates instead, as I'm pretty sure it's not in line with the design system guidelines. I'll ask UXers about this. But until then I guess we'll have to use custom components.